tmail-backend/apps/memory/src/main/java/com/linagora/tmail/james/app/MemoryServer.java

```diff
@@ -38,6 +46,18 @@
 import com.linagora.tmail.james.jmap.ticket.TicketRoutesModule;
 
 public class MemoryServer {
+    public static final Module IN_MEMORY_SERVER_MODULE = Modules.combine(
+        new CustomMailetProcessingModule(),
+        new BlobMemoryModule(),
+        new DeletedMessageVaultModule(),
+        new BlobExportMechanismModule(),
```

Comment:
Is that one necessary? (and the DeletedMessageVaultModule above too?)

Reply:
Clean up should come in a follow up PR (they were here before, their removal is an enhancement)

Technically speaking DTM is a feature of OpenPaaS thus its support is compulsory in TMail.